style.css

```diff
@@ -0,0 +1,126 @@
+body {
+  width: 375px;
+  height: 587.15px;
+}
+
+.logoholder {
+  order: 0;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.menuholder {
+  order: 0;
+  margin-right: 24px;
+  width: 40px;
+  height: 40px;
+  margin-top: 20px;
+}
+
+.mylogo {
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin-left: 24px;
+  margin-top: 26px;
+  text-decoration: none;
+}
+
+.headerholder {
+  width: 327px;
+  height: 150px;
+  margin-left: 24px;
+  margin-top: 114px;
+}
+
+.head {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  order: 0;
+  margin-top: auto;
+  margin-bottom: auto;
+  width: 327px;
+  height: 150px;
+  position: static;
+}
+
+.headtext {
+  width: 327px;
+  height: auto;
+  left: 24px;
+  margin-left: 24px;
+  margin-right: 24px;
+  background-color: white;
+  font-family: Poppins, sans-serif;
```

Comment:
- [ ] It would be better if we implement the CSS DRY principle, for example; ` font-family: Poppins, sans-serif;` has been repeated